code/game/atoms_movable.dm

```diff
@@ -129,6 +133,12 @@
 
 	for(var/datum/light_source/L in light_sources)
 		L.source_atom.update_light()
+	
+	for(var/mob/M in GetAllContents()) // Update parallax of contents
+		if(M.hud_used)
+			M.hud_used.update_parallax()
+	if(ismob(src) && src:hud_used)

```

Comment:
Define mob/forceMove and call the parent, never override type saftey with colon operators.